File: Exercícios sobre POO/Carro 1-2-6/Carro em python/pneu.py
class Pneu:
    def __init__(self, marca, tamanho) -> None:
        self.marca = marca
        self.tamanho = tamanho

    def inflar(self):
        print("O pneu esta inflado")

    def desinflar(self):
        print("O pneu esta desinflado")

File: Exercícios sobre POO/Carro 1-2-6/Carro em java/src/Pneu.java
public class Pneu {
    private String marca;
    private double tamanho;
    public Pneu(String marca, int tamanho) {
        this.marca = marca;
        this.tamanho = tamanho;
    }
    public void inflar(){
        System.out.println("O pneu esta inflado");
    }
    public void desinflar(){
        System.out.println("O pneu esta desinflado");
    }
}